api/go.mod

```diff
@@ -0,0 +1,22 @@
+module kubevirt.io/vm-console-proxy/api
+
+go 1.22.4
+
+require k8s.io/apimachinery v0.30.3
```

Comment:
Any reason why this lives in its own require statement?

Reply:
The below `require` block was generated by `go mod tidy && go mod vendor` and contains only `// indirect` dependencies.

Reply:
Ah yes, I missed it. Disregard that comment.